apps/src/code-studio/components/progress/SectionSelector.jsx

```diff
@@ -29,18 +30,18 @@ const SectionSelector = React.createClass({
     ).isRequired,
     selectedSectionId: PropTypes.string,
     selectSection: PropTypes.func.isRequired,
-  },
+  }
```

Comment:
tiny nit: do you need a comma here?

Reply:
You're right, I should.  Not a thing our linter catches yet :grin: